src/site/resources.liquid

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
this file has been updated quite a bit: https://github.com/philhawksworth/the-united-effort-orginization/blob/main/src/site/static/resources.liquid